packages/origin-ui/src/components/Header.tsx

```diff
@@ -19,23 +19,38 @@ import { NavLink, withRouter, RouteComponentProps } from 'react-router-dom';
 import { User } from '@energyweb/user-registry';
 import logo from '../../assets/logo.svg';
 import { AccountCircle } from '@material-ui/icons';
+import { Configuration } from '@energyweb/utils-general';
 
 import './Header.scss';
 import { connect } from 'react-redux';
 import { IStoreState } from '../types';
-import { getBaseURL, getCurrentUser } from '../features/selectors';
+import { getBaseURL, getCurrentUser, getConfiguration } from '../features/selectors';
 import { getAssetsLink, getCertificatesLink, getDemandsLink } from '../utils/routing';
+import { AccountDetailsModal } from '../elements/Modal/AccountDetailsModal';
 
-interface IStateProps {
+interface IHeaderProps {
```

Comment:
Could we keep the convention to name state properties (properties coming from Redux state) `IStateProps`?

If component has its own props then IMO it should be called `IOwnProps`.

`IHeaderProps` is all of the properties, which in this case would be:
```
type Props = RouteComponentProps & IStateProps;
```